docs/kpod-inspect.1.md

```diff
@@ -28,144 +28,72 @@ Display the total file size if the type is a container
 
 ## EXAMPLE
 
-kpod inspect redis:alpine
-
-{
-    "ArgsEscaped": true,
-    "AttachStderr": false,
-    "AttachStdin": false,
-    "AttachStdout": false,
-    "Cmd": [
-	"/bin/sh",
-	"-c",
-	"#(nop) ",
-	"CMD [\"redis-server\"]"
-    ],
-    "Domainname": "",
-    "Entrypoint": [
-	"entrypoint.sh"
-    ],
-    "Env": [
-	"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
-	"REDIS_VERSION=3.2.9",
-	"REDIS_DOWNLOAD_URL=http://download.redis.io/releases/redis-3.2.9.tar.gz",
-	"REDIS_DOWNLOAD_SHA=6eaacfa983b287e440d0839ead20c2231749d5d6b78bbe0e0ffa3a890c59ff26"
-    ],
-    "ExposedPorts": {
-	"6379/tcp": {}
-    },
-    "Hostname": "e1ede117fb1e",
-    "Image": "sha256:75e877aa15b534396de82d385386cc4dda7819d5cbb018b9f97b77aeb8f4b55a",
-    "Labels": {},
-    "OnBuild": [],
-    "OpenStdin": false,
-    "StdinOnce": false,
-    "Tty": false,
-    "User": "",
-    "Volumes": {
-	"/data": {}
-    },
-    "WorkingDir": "/data"
-}
+```
+# kpod inspect umohnani/get-started:part1
```

Comment:
you may want to consider changing the 'umohani' to something generic like 'myusername' or some such.

Reply:
or perhaps doing redis:alpine again, an image people would be able to easily play with on their system.

Reply:
Wanted to use an image that had a bunch of things configured like ports and all, but using fedora now. fixed